src/foundry/common/documents.mjs/baseNote.d.ts

```diff
@@ -1,10 +1,13 @@
+import { ConfiguredDocumentClass } from '../../../types/helperTypes';
 import { DocumentMetadata } from '../abstract/document.mjs';
 import { Document } from '../abstract/module.mjs';
+import * as data from '../data/data.mjs';
+import { BaseScene } from './baseScene';
 
 /**
  * The base Note model definition which defines common behavior of an Note document between both client and server.
  */
-export declare class BaseNote extends Document<any, any> {
+export declare class BaseNote extends Document<data.NoteData, InstanceType<ConfiguredDocumentClass<typeof BaseScene>>> {
```

Comment:
This is missing `get schema()`, `get metadata()` is incorrect, and the `testUserPermission` override is missing.